src/executables/ddl_proxy_client.cpp

```diff
@@ -101,6 +102,11 @@ main (int argc, char *argv[])
 	  break;
 
 	case DDL_PROXY_COMMAND_S:
+	  if (arguments.do_extract_schema)
+	    {
+	      // TODO : incompatible args : print usage
```

Comment:
is proxy ddl supposed to have a usage? isn't it internal only?

Reply:
http://jira.cubrid.org/browse/CBRD-23213

Reply:
It is an executable delivered in $CUBRID/bin/. There is nothing preventing user to run it. 
http://jira.cubrid.org/browse/CBRD-22876

Reply:
that is true, but is it desirable to make it visible? even though it has access on CUBRID/bin executable, user will not use it if we don't include it in manual/command list.

Reply:
The previously mentioned jira issue will deal with this topic.